mergify_engine/web/config_validator.py

```diff
@@ -70,3 +72,13 @@ async def config_validator(
         message = "The configuration is valid"
 
     return responses.PlainTextResponse(message, status_code=status)
+
+
+@app.exception_handler(engine_exceptions.RateLimited)
```

Comment:
do we have to repeat this in all blueprints?

Reply:
This is not a blueprint (router in Scarlette/FastAPI wording), but a full app.

We have multiples apps instead of multiple routers to not set up CORS on all endpoints.